runsharp/full_gui.py

```diff
@@ -538,7 +541,11 @@ def skewApp(self, filename=None, ntry=0):
             else:
                 logging.debug("Data was found and successfully decoded!")
                 prof_collection = ret[0]
-        
+            print prof_collection._profs
```

Comment:
This line looks like debug code that should be removed.

Reply:
Okay, I think I fixed that.  It's in the latest commit.